src/sync/index.js

```diff
@@ -283,6 +286,8 @@ async function syncDecentralizedOracleCreated(db, startBlock, endBlock, removeHe
 
             decentralOracle.name = topic.name;
             decentralOracle.options = topic.options;
+            decentralOracle.startTime = currentBlockTime;
```

Comment:
we can use utc.now() or even leave it blank if not provided

Reply:
`utc.now()` would not be accurate since that would take the system time (or more like the time the user was parsing this block). By nature it should always be available, since we are parsing a past block, that block will always have a blockTime.